versioned/3.1.0/jiva-operator-RC1.yaml

```diff
@@ -0,0 +1,4053 @@
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openebs
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: jivavolumepolicies.openebs.io
+spec:
+  group: openebs.io
+  names:
+    kind: JivaVolumePolicy
+    listKind: JivaVolumePolicyList
+    plural: jivavolumepolicies
+    shortNames:
+    - jvp
+    singular: jivavolumepolicy
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
```

Comment:
The crds are outdated. Can you please take reference this operator yaml https://github.com/openebs/jiva-operator/blob/develop/deploy/jiva-operator.yaml